src/Appwrite/Migration/Version/V06.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Appwrite\Migration\Version;
+
+
+use Utopia\App;
+use Utopia\CLI\Console;
+use Appwrite\Database\Database;
+use Appwrite\Database\Document;
+use Appwrite\Migration\Migration;
+use Appwrite\OpenSSL\OpenSSL;
+
+class V06 extends Migration
+{
+    public function execute(): void
+    {
+        $project = $this->project;
+        Console::log('Migrating project: ' . $project->getAttribute('name') . ' (' . $project->getId() . ')');
+
+        $this->projectDB->disableFilters();
+        $this->forEachDocument([$this, 'fixDocument']);
+        $this->projectDB->enableFilters();
+    }
+
+    protected function fixDocument(Document $document)
+    {
+        switch ($document->getAttribute('$collection')) {
+            case Database::SYSTEM_COLLECTION_USERS:
+                if ($document->getAttribute('password-update', null)) {
+                    $document
+                        ->setAttribute('passwordUpdate', $document->getAttribute('password-update', $document->getAttribute('passwordUpdate', '')))
+                        ->removeAttribute('password-update');
+                }
+                break;
+            case Database::SYSTEM_COLLECTION_KEYS:
+                if ($document->getAttribute('secret', null)) {
```

Comment:
I would add a small check here to test if the key has already been encrypted properly in case someone runs the migration script multiple times.